File: ws/chavez/u3/ws26Interfaces/AnimalFarmSystem/src/main/java/ec/edu/espe/animalfarmsystem/model/SlaughterHouse.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.animalfarmsystem.model;

/**
 *
 * @author Odalys Chavez , CodeBreakers, @ESPE
 */
public class SlaughterHouse {
    private int id;
    private String name;
    private String address;
    private String cellphoneNumber;

    public SlaughterHouse(int id, String name, String address, String cellphoneNumber) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.cellphoneNumber = cellphoneNumber;
    }

    @Override
    public String toString() {
        return "SlaugterHouse{" + "id=" + id + ", name=" + name + ", address=" + address + ", cellphoneNumber=" + cellphoneNumber + '}';
    }
    

    /**
     * @return the id
     */
    public int getId() {
        return id;
    }

    /**
     * @param id the id to set
     */
    public void setId(int id) {
        this.id = id;
    }

    /**
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name the name to set
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * @return the address
     */
    public String getAddress() {
        return address;
    }

    /**
     * @param address the address to set
     */
    public void setAddress(String address) {
        this.address = address;
    }

    /**
     * @return the cellphoneNumber
     */
    public String getCellphoneNumber() {
        return cellphoneNumber;
    }

    /**
     * @param cellphoneNumber the cellphoneNumber to set
     */
    public void setCellphoneNumber(String cellphoneNumber) {
        this.cellphoneNumber = cellphoneNumber;
    }
    
    
}

File: hw/tipantiza/u3/hw21FarmSystemPOO/FarmSystem_js/model/slaughter-house.js
export class SlaughterHouse {
    constructor(id, name, address, cellphoneNumber) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.cellphoneNumber = cellphoneNumber;
    }
    
    toString() {
        return `SlaughterHouse(id=${this.id}, name=${this.name}, address=${this.address}, cellphoneNumber=${this.cellphoneNumber})`;
    }
    
    getId() {
        return this.id;
    }
    
    setId(id) {
        this.id = id;
    }
    
    getName() {
        return this.name;
    }
    
    setName(name) {
        this.name = name;
    }
    
    getAddress() {
        return this.address;
    }
    
    setAddress(address) {
        this.address = address;
    }
    
    getCellphoneNumber() {
        return this.cellphoneNumber;
    }
    
    setCellphoneNumber(cellphoneNumber) {
        this.cellphoneNumber = cellphoneNumber;
    }
}